pytorch/src/main/java/org/bytedeco/pytorch/presets/torch_cuda.java

```diff
@@ -48,6 +48,12 @@
                 "ATen/cuda/CUDAGeneratorImpl.h"
             }
         ),
+        @Platform(
+                value = "windows",
+                extension = "-gpu",
+                link = { "cudart" },
+                linkpath    = { "C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v12.1/lib/x64/" }
+        ),
```

Comment:
This is going to be necessary on Linux as well, it just won't complain until you try to call a function.

Reply:
The linux JNI was already linked with cudart, probably because the linker finds out the needed symbols and where to find them according to transitive dependencies. But it doesn't hurt to explicitly specify it.
Anyway this didn't work on windows and the job you relaunched failed again on undefined symbol. I don't know why. Another attempt running.